### ISSUE TITLE
Single WAR fails to start if it contains libs with Spring components

### ISSUE THREAD

**Issue by haulmont-git** [cuba-platform/cuba#454]
1. Create new project using Studio
1. Add dependencies to the global module with spring components. For example: 

```
compile('io.springfox:springfox-swagger2:2.7.0')
compile('io.springfox:springfox-swagger-ui:2.7.0')
```

1. Build single WAR
1. Try to start single WAR on the Tomcat
Error occured

---
Original issue: https://youtrack.haulmont.com/issue/PL-9201

**Comment by andreysubbotin** [comment]
@soraksh, please test that it works with new single WAR deployment

**Comment by soraksh** [comment]
Should be fixed with #2060
